examples/ml/rnn/pipeline-combined.json

```diff
@@ -24,7 +24,7 @@
     "name": "GoT_generate"
   },
   "transform": {
-    "image": "tensor_flow_rnn_got",
+    "image": "nickharvey/tensorflow_0_10_rnn_got:v1",
```

Comment:
Probably shouldn't use your personal dockerhub account here. Could you push it to `pachyderm/tensorflow_0_10_rnn_got:v1`